examples/accordion/accordion.html

```diff
@@ -38,30 +38,17 @@ <h2 id="ex_label">Example</h2>
     <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
     <div id="coding-arena">
       <div class="demo-block">
-        <!-- Accordion Configuration Options
-
-          data-allow-toggle
-            Allow for each toggle to both open and close individually
-
-          data-allow-multiple
-            Allow for multiple accordion sections to be expanded at the same time. Assumes data-allow-toggle otherwise you would not be able to close any of the accordions
-          __________
-
-          Ex:
-            <dl id="accordionGroup" role="presentation" class="Accordion" data-allow-multiple>
-
-            <dl id="accordionGroup" role="presentation" class="Accordion" data-allow-toggle>
-
-        -->
-        <dl id="accordionGroup" role="presentation" class="Accordion">
-          <dt role="heading" aria-level="3">
+        <div id="accordionGroup" role="presentation" class="Accordion">
+          <h3>
             <button aria-expanded="true" class="Accordion-trigger"
-              aria-controls="sect1" id="accordion1id" type="button"
-            >
-              <span class="Accordion-title">Personal Information</span><span class="Accordion-icon"></span>
+              aria-controls="sect1" id="accordion1id">
+              <div class="Accordion-title">
```

Comment:
What is reason for changing from span to div? The validator is failing this; says div not allowed in this context as child of button.